desc/protoparse/parser.go

```diff
@@ -104,6 +104,14 @@ type Parser struct {
 	// with import statements all being relative to the root of this tree).
 	InferImportPaths bool
 
+	// FileDescriptors is a map of filenames to file descriptor protos, which
+	// will be consulted when resolving imports. This allows a compiled Go
+	// proto in another Go module to be referenced in the proto(s) being parsed.
+	//
+	// In order to get file descriptors from compiled Go protos, use the
+	// GetRegisteredDescriptors method.
```

Comment:
Let's remove this paragraph. (See related comment below on `GetRegisteredDescriptors`.)

Reply:
I changed it to reference `LoadFileDescriptor`. (This was sufficiently non-obvious to me that I expect I will not be the only person who will wonder how to do this.)

Reply:
(No longer relevant.)